<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuideDlg</class>
 <widget class="QDialog" name="GuideDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导航</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="newBtn">
       <property name="text">
        <string>新游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadBtn">
       <property name="text">
        <string>加载存档</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mgBtn">
       <property name="text">
        <string>后台管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitBtn">
       <property name="text">
        <string>退出游戏</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>26</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
